dlme_airflow/utils/ifpo_get_thumbnail_urls.py

```diff
@@ -11,19 +11,23 @@ def get_thumbnail_url(id):
     req = requests.get(url)
     return f"{req.url}/large"
 
-def add_thumbnail_urls():
+def add_thumbnail_urls(**kwargs):
     # Fetch working directory path from catalog and read file into Pandas dataframe
     catalog = catalog_for_provider('ifpo')
     root_dir = os.path.dirname(os.path.abspath('metadata'))
-    logging.info(f'Root directory path is {root_dir}.')
     data_path = catalog.metadata.get('data_path', 'ifpo/photographs')
-    logging.info(f'Data path is {root_dir}.')
     working_csv = os.path.join(root_dir, 'working', data_path, 'data.csv')
-
     df = pd.read_csv(working_csv)
 
-    # Build thumbnail url and use that to fetch larger image from redirect
-    df['thumbnail'] = ['test']
+    # A column can only be added to a data from of the same length, build an array
+    # for thumbnails that is the appropriate length
+    array_of_thumbnails = ['test'] * len(df.index)  # This is just an example with the value 'test'
```

Comment:
You are welcome to merge at this point, however, note that this is where I think the meat of your incoming changes will be, building this array properly. i think this PR gets you to a place where that can happen - for now, it adds a column to the csv completely populated with `test`.